references/organization/_06-Project-Plan-and-Workflow.md

```diff
@@ -0,0 +1,64 @@
+The conclusion from this is that even though the outcome model is the same for the data generation process, we can only assign causal interpretations to parameters if we have clearly drawn our assumptions on the data generating process. Without those, one may easily challenge any interpretation we assign to our parameters by simply asking the question “but what if the data generating process looked like this?”. By drawing a causal graph upfront, we limit this sort of questions to questions about the validity of one’s causal graph, which comes with testable implications and create much more interesting and focused discussions than those created when there’s no causal graph that clearly shows the modeler’s assumptions. 
+
+##### Concrete steps:
+
+Functions <br>
+-	The main function we need is one that takes as input a hypothesized causal graph (through a set of nodes and edges), with distributional assumptions and parametric relationships between variables, and simulates data given those inputs. 
+-	Since we’re dealing with a multinomial choice problem, we need a function to simulate outcomes and be flexible to take in multiple specifications for each utility equation.
+
+Workflow <br>
+-	Come up with an outcome (choice) model based on the bike data in Brathwaite (2018). The model doesn’t need to be the exact model estimated in the paper, but that can be the case. We just need to select a set of variables while being mindful that one of those variables will be used as a latent confounder in the next step. 
+-	Simulate two sets of data for the X variables: <br>
+    - One where all the Xs are independent (i.e. the only edges in the causal graph are between the X and the outcome variable). 
+    - One based on a realistic causal graph with confounders. Preferably focus on a case with one latent confounder (like distance? Age? Gender?) <br>
+-	For each of the two causal graphs above, simulate choice data based on the outcome model assumed. <br>
+-	Estimate the choice model for each of those two datasets. <br>
+-	Repeat the process n times, and calculate the percentage of those times where the estimated parameters’ confidence intervals contain the true parameter. <br>
+
+### Dealing with latent confounders in the causal graph
+
+In this part of the work, we take the realistic causal graph from part 1 and remove a latent confounder from it. We then attempt to recover the latent confounder using the de-confounder algorithm proposed by Blei et al. (2018). We will apply this work to the simulated data set, where we know the true parameters of interest. We will repeat the same simulation process in part 1 where we sample from the population and estimate models where we control for the latent confounder using the deconfounder approach and look at the percent of time we recover the true parameter. 
```

Comment:
nit: "[...] remove a confounder from the dataset, making the confounder latent with respect to the graph."